core-kotlin-modules/core-kotlin-collections-map/src/test/kotlin/com/baeldung/removekey/RemoveKeyDuringIterationUnitTest.kt

```diff
@@ -0,0 +1,63 @@
+package com.baeldung.removekey
+
+import org.junit.jupiter.api.Test
+import java.util.ConcurrentModificationException
+import kotlin.test.assertEquals
+import kotlin.test.assertFailsWith
+
+class RemoveKeyDuringIterationUnitTest {
```

Comment:
Let's rename this to better reflect what is going on. We remove entries from the map using their keys, rather than removing keys.

Similar changes should be make to the package name, and the test names.

```suggestion
class RemoveMapEntryDuringIterationUnitTest {
```

Reply:
Perfect! The new class name makes more sense now. Thanks for suggesting!